docs/installation/virtual/proxmox.md

```
@@ -37,13 +39,28 @@ Proxmox is an open-source platform for virtualization.
    $ qm set 200 --ide2 local-lvm:cloudinit
    ```
 
+   Configure the Cloud-Init user data before the first boot. For key-based
+   access, create the `vyos` user and import public SSH keys from a file on the
+   Proxmox host:
+
+   ```bash
+   $ qm set 200 --ciuser vyos
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)